cpp/autosar/src/rules/A0-1-3/UnusedLocalFunction.ql

```diff
@@ -19,6 +19,44 @@ import codingstandards.cpp.autosar
 import codingstandards.cpp.DynamicCallGraph
 import codingstandards.cpp.deadcode.UnusedFunctions
 
+/** Checks if a function call exists to the function
+ * passed in the arguments.
+ */
+predicate isCalled(Function unusedFunction) {
+  exists (FunctionCall f | unusedFunction.getACallToThisFunction() = f)
```

Comment:
Use the `DynamicCallGraph::getTarget` predicate, like so:
```suggestion
  unusedFunction = getTarget(_)
```

This predicate includes everything reported here, plus includes additional logic for resolving across linker boundaries and dispatch. I don't know if it's strictly necessary, given the restriction in this rule to local functions, but it is consistent with what we already use in the query itself and should not be harmful.